rook/kubestone_iperf/bootstrap.py

```diff
@@ -0,0 +1,69 @@
+import os
+import yaml
+
+
+config_dir = './kubestone_iperf'
+
+
+iperf_defination = {
```

Comment:
is `defination` a typo?

Reply:
Turns out this type was places in many more places. Fixed all of them.